client/src/components/application/NonlinearStepper/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Container, ThemeProvider } from '@material-ui/core';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepButton from '@material-ui/core/StepButton';
+import ProInfos from './ProInfos';
+import GeneralInfos from './GeneralInfo';
+import { Theme } from '../../common/Typography/style';
```

Comment:
this import stmt, I prefer to be before the end one, useStyles 
Just kind of code arrangement